test/controllers/api/v2/api_controller_test.rb

```diff
@@ -21,6 +21,20 @@ def teardown
       @options = nil
     end
 
+    def test_check_katello_agent_not_disabled
+      ::Katello.expects(:with_katello_agent?).returns(true)
+      @controller.expects(:fail).never
+
+      @controller.check_katello_agent_not_disabled
+    end
+
+    def test_check_katello_agent_not_disabled_fail
+      ::Katello.expects(:with_katello_agent?).returns(false)
+      @controller.expects(:fail).with(HttpErrors::BadRequest, "This action uses katello-agent, which is currently disabled in Settings.")
```

Comment:
@chris1984 Is "disabled in Settings" the correct terminology here? Can the user change this setting on the Settings page, or does it require an installer run?

Reply:
it's an installer setting. would advocate wording that doesnt encourage enabling it, but rather using REX